<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawWidget</class>
 <widget class="QWidget" name="drawWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="gCodeEditWidget" name="gCodeTextEdit" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="viewWidget" name="vWidget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>901</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>811</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки</string>
   </property>
   <widget class="QDoubleSpinBox" name="zPosSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Положение по Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>31</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина пера</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="penSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="reParseButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Пересчитать программу с новыми параметрами.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Пересчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>31</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость подачи</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="forceSpinBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="moveSpinBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>31</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Скорость перемещения</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>450</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Пуск</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>450</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="showRadioButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>390</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отображение рисунка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Отобразить</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="simulateRadioButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>410</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Симуляция отрисовки с реальной скоростью интрумента.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Симуляция</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="workRadioButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>430</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отрисовка на реальном плоттере.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Работа</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>viewWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/viewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gCodeEditWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/gcodeeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
